pyro/distributions/categorical.py

```diff
@@ -41,6 +43,7 @@ def __init__(self, ps=None, vs=None, logits=None, batch_size=None, log_pdf_mask=
         self.vs = self._process_data(vs)
         self.log_pdf_mask = log_pdf_mask
         if vs is not None:
+            warnings.warn('Categorical vs argument is deprecated', UserWarning)
```

Comment:
Are there existing examples/tutorials that use this argument currently? If not, we might as well just remove it on `dev`, as it will be removed by the next release.

Reply:
I believe `vs` is not used in any examples or tutorials. I looked into removing `vs` code but that turned out to be more work than adding a simple warning :smile:

Reply:
Looks good! We can definitely get this merged as is, and remove it completely along with the tests soon.